### PR TITLE
Snake game using Python

### LINKED ISSUE #295: Snake Game in Python
# AIM 

- Create a Snake Game using Python

# Details 

- Game Over when Snake hits the boundaries.
- Adding the Food.
- Increasing the Length of the Snake as Snake Eats the food
- Display the Score.

### PR DESCRIPTION
**Pull request by CharvyJain**
# Description

Added the Script of Snake Game using Pygame

## Fixes #295

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [ ] Yes

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

### CODE REVIEW

**Review by ShristiJalan — APPROVED** [review]

File: Snake Game (GUI)/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+In order to run this script, You just need the following 3 modules -
+
+- **Pygame:** It is a set of Python modules designed for writing video games.
+- **Time:** This function is used to count the number of seconds elapsed since the epoch.
+- **Random:** This function is used to generate random numbers in Python by using random module. **Pygame, Time and Random**
+
+## Output
+
+![Snake Game](game.png)
```
Comment (avinashkranjan) [inline Snake Game (GUI)/README.md:15]: Add Image Rather than link

**Review by avinashkranjan — CHANGES_REQUESTED** [review]
Rest LGTM..🚀

**Review by atarax665 — APPROVED** [review]

File: Snake Game (GUI)/README.md (hunk 1 above)
Comment (CharvyJain) [reply Snake Game (GUI)/README.md:15]: Sorry, I didn't get you!

**Review by avinashkranjan — APPROVED** [review]

**Comment by avinashkranjan** [comment]
@all-contributors please add @CharvyJain  for code and documentation

**Pull request merged**